client/api_sp.go

```diff
@@ -240,14 +322,29 @@ func (c *Client) UpdateSpStoragePrice(ctx context.Context, spAddr string, readPr
 		StorePrice:    storePrice,
 		FreeReadQuota: freeReadQuota,
 	}
-	resp, err := c.chainClient.BroadcastTx(ctx, []sdk.Msg{msgUpdateStoragePrice}, &TxOption)
+	resp, err := c.chainClient.BroadcastTx(ctx, []sdk.Msg{msgUpdateStoragePrice}, &txOption)
 	if err != nil {
 		return "", err
 	}
 	return resp.TxResponse.TxHash, nil
 }
 
-func (c *Client) UpdateSpStatus(ctx context.Context, spAddr string, status spTypes.Status, duration int64, TxOption gnfdSdkTypes.TxOption) (string, error) {
+// UpdateSpStatus -Set an SP status between STATUS_IN_SERVICE and STATUS_IN_MAINTENANCE.
+//
```

Comment:
Can we add more details in which case users need to use this API?

Reply:
added